layouts/shortcodes/pages/newsletter.html

```diff
@@ -1,189 +1,42 @@
-<div class="panel-group margin-top-30" id="accordion" role="tablist" aria-multiselectable="true">
-  <div class="panel panel-default">
-    <div class="panel-heading" role="tab" id="heading2020">
-      <h2 class="panel-title">
-        <a role="button" data-toggle="collapse" data-parent="#accordion" href="#collapse2020" aria-expanded="true" aria-controls="collapse2020">
-        2020
-        </a>
-      </h2>
-    </div>
-    <div id="collapse2020" class="panel-collapse collapse in" role="tabpanel" aria-labelledby="heading2020">
-      <div class="panel-body">
-        <div class="row margin-bottom-15">
-          <div class="col-sm-12">
-            <div class="col-md-8 col-sm-6">
-              <span class="newsletter-archive-icons">
-              <i class="fa fa-calendar-o" aria-hidden="true"></i>
-              <span class="newsletter-archive-month">May</span>
-              </span>
-            </div>
-            <div class="col-md-16 col-sm-18">
-              <p><a href="https://www.eclipse.org/community/eclipse_newsletter/2020/may/" target="_blank">Jakarta EE 9 and Beyond
-                </a>
-              </p>
-            </div>
-          </div>
-        </div>
-        <hr />
-        <div class="row">
-          <div class="col-sm-12">
-            <div class="col-md-8 col-sm-6">
-              <span class="newsletter-archive-icons">
-              <i class="fa fa-calendar-o" aria-hidden="true"></i>
-              <span class="newsletter-archive-month">February</span>
-              </span>
-            </div>
-            <div class="col-md-16 col-sm-18">
-              <p><a href="https://www.eclipse.org/community/eclipse_newsletter/2020/february/" target="_blank">Jakarta EE: The Next Phase of Evolution
-                </a>
-              </p>
-            </div>
-          </div>
-        </div>
-      </div>
+<h2 class="big-text-secondary margin-bottom-20">Jakarta EE Newsletters Archives</h2>
+
+<div class="row margin-bottom-40">
+  <div class="col-sm-8 card-container">
+    <div class="card-panel bordered panel panel-default match-height-item-by-row">
+      <h3 class="big-text text-secondary"><i class="fa fa-calendar-o" aria-hidden="true"></i> 2020</h3>
+      <ul>
+        <li>May: <a href="https://www.eclipse.org/community/eclipse_newsletter/2020/may/" target="_blank">Jakarta EE 9 and Beyond</a></li>
```

Comment:
We should consider making the month clickable too just for convenience sake. WDYT?

Reply:
Sure, probably better to make the whole sentence clickable.